server/api/handlers/new_game.go

```diff
@@ -54,3 +73,12 @@ func ProcessNewGameRequest(context context.Context, req *ws_interface.ApiRequest
 
 	return toGameSessionResponse(session), nil
 }
+
+func getGameListedInCasino(gameId uint64, casGames *[]*models.CasinoGame) *models.CasinoGame {
```

Comment:
is pointer needed here?

Reply:
where exactly ? arguments or returning ?
in arguments need to avoid copying, in return value by same reasons

Reply:
where exactly ? arguments or returning ?
in arguments need to avoid copying, in return value by same reasons

Reply:
fixed